src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/Design/DesigntimeLicenseContext.cs

```diff
@@ -79,13 +79,13 @@ public override string GetSavedLicenseKey(Type type, Assembly resourceAssembly)
                     {
                         // Though, I could not repro this, we seem to be hitting an AssemblyBuilder
                         // when walking through all the assemblies in the current app domain. This throws an
-                        // exception on Assembly.CodeBase and we bail out. Catching exceptions here is not a
+                        // exception on Assembly.Location and we bail out. Catching exceptions here is not a
                         // bad thing.
                         if (asm.IsDynamic)
                             continue;
 
                         // file://fullpath/foo.exe
-                        string fileName = GetLocalPath(asm.EscapedCodeBase);
+                        string fileName = GetLocalPath(asm.Location);
```

Comment:
This should be just `string fileName = asm.Location;`; and the file:// comment should be deleted.

Reply:
This also needs a check for `Assembly.Location` returning empty string:
* empty string is returned by assemblies loaded from memory - and will also be used by assemblies in a single file bundle.
* The code below will throw if this happens

Reply:
Updated based on feedback